torch/_utils_internal.py

```diff
@@ -33,5 +34,23 @@ def resolve_library_path(path):
     return os.path.realpath(path)
 
 
+def get_source_lines_and_file(obj):
+    """
+    Wrapper around inspect.getsourcelines and inspect.getsourcefile.
+
+    Returns: (sourcelines, file_lino, filename)
+    """
+    filename = None  # in case getsourcefile throws
+    try:
+        filename = inspect.getsourcefile(obj)
+        sourcelines, file_lineno = inspect.getsourcelines(obj)
+    except OSError as e:
+        raise OSError((
+            "Can't get source for {}. TorchScript requires source access in order to carry out compilation. " +
+            "Make sure original .py files are available. Original error: {}").format(filename, e))
```

Comment:
Won't `filename` usually be `None` when this error is thrown? Maybe it should print the `obj` as well